baseframe/forms/validators.py

```diff
@@ -379,7 +390,7 @@ def __call__(self, form, field):
         if diagnosis.code == 0:
             return
         else:
-            raise StopValidation(self.message or _(diagnosis.message))
+            raise StopValidation(self.message or __(diagnosis.message))
```

Comment:
This is at runtime, so it should use `_` (single underscore). The difference is this:

1. `_` (alias for `gettext`) does an immediate translation to whatever locale is detected from the request. However, if there is no request, it falls back to the default, which is `en`. This means it cannot be used at the module level or class level, as those statements are executed at program load time.

2. `__` (alias for `lazy_gettext`) creates a "lazy string". It is not an actual string, but a class that does the translation whenever it is cast into a string. Assuming that such casting is only done in templates, it is safe to create lazy strings at program load time. It is a safe substitute for `_`, but since it is relatively inefficient, it should not be used when `_` will suffice.